packages/realm/src/app-services/App.ts

```diff
@@ -119,6 +119,7 @@ export type AppConfiguration = {
   baseUrl?: string;
 
   /**
+   * @deprecated LocalAppConfiguration is no longer used by Atlas Device Sync. It will be removed from in future SDK releases.
```

Comment:
No need to mention the symbol in its documentation.

```suggestion
   * @deprecated No longer used by Atlas Device Sync. It will be removed from in future SDK releases.
```